tests/behat/features/asp/ondemand-unauth.feature

```diff
@@ -64,21 +64,22 @@ Feature: This Behat test checks the OnDemand page
     Then I should see "Learn more about OnDemand"
     Then I should see an image with alt text "Laptop team"
     Then I should see "ACCESS OnDemand helps computational researchers and students efficiently"
-    #TODO Need to test link to Learn More. There are two Learn mre buttons
+    # test Learn More, including link
     Then I should see "Learn More"
+    When I go to "http://openondemand.org/"
+    Then I should get a "200" HTTP response
+    When I go to "/ondemand"
```

Comment:
Can we use the a[href=] here?

Reply:
In this case, I think it makes more sense to say `When I am on "/ondeman"` -- thoughts, @a-pasquale ?

Reply:
There are 242 instances of "When I go to ..." in the tests.  I think I should make a pass at updating all of those - agreed?

Reply:
update: 
when testing things on a specific page, it makes sense to start by saying `When I am on ...`
but when testing for the existance of a link it makes sense to say `Then I should see an a[href='http://openondemand.org/']" element`

Reply:
This makes sense to me, that's what I am saying also. I'm not sure what you meant by the previous comment.

Reply:
e.g., line 69 should be Then I should see an "a[href='http://openondemand.org/']" element, but line 71 should be When I am on "/ondemand"